sklearn/metrics/_ranking.py

```diff
@@ -539,7 +539,7 @@ def _binary_clf_curve(y_true, y_score, pos_label=None, sample_weight=None):
         classes_repr = ", ".join(repr(c) for c in classes)
         raise ValueError("y_true takes value in {{{classes_repr}}} and "
                          "pos_label is not specified: either make y_true "
-                         "take integer value in {{0, 1}} or {{-1, 1}} or "
+                         "take value in {{0, 1}} or {{-1, 1}} or "
```

Comment:
single brackets?

Reply:
sorry what do you mean?

Reply:
{} instead of {{}}

Reply:
@NicolasHug We can't use {} here because {} has special meaning. We need to use ``{{}}`` to output ``{}``.